packages/core/src/components/viewer-grid/viewer-grid.stories.ts

```diff
@@ -1,7 +1,10 @@
-import { storiesOfViewerGrid } from "@dso-toolkit/sources";
+import { storiesOfViewerGrid, Tile } from "@dso-toolkit/sources";
 import { storiesOf } from "@storybook/web-components";
 import { html } from "lit-html";
 
+import { tileGridTemplate } from '../../../../../packages/css/src/components/tile-grid/tile-grid.template';
```

Comment:
@benheldoorn FYI: We kunnen op deze obscure manier vanaf de `core` package HTML/CSS templates uit de `css` package importeren.. Niet fraai, maar beter dan code dupliceren. Binnenkort zijn `core` en `css` één storybook en hoeven we er niet meer over na te denken.